### PR TITLE
AC logs in debug dir are now per trigger, inside subdir

### PR DESCRIPTION
**Pull request by maloel**

### CODE REVIEW

File: src/l500/ac-trigger.cpp [hunk 1]
```diff
@@ -437,26 +446,107 @@ namespace ivcam2 {
...
+
+            // If the user has this env var defined, then we write out logs and frames to it
+            // NOTE: The var should end with a directory separator \ or /
+            auto dir_ = getenv( "RS2_DEBUG_DIR" );
+            if( dir_ )
+            {
+                path = dir_;
+                add_dir_sep( path );
+
+                std::time_t now = std::time( nullptr );
```
Comment (Nir-Az) [inline src/l500/ac-trigger.cpp]: Consider reuse now_string() function. maybe with a default parameter of now_string(bool add_date = false)

File: src/l500/ac-trigger.cpp [hunk 2]
```diff
@@ -437,26 +446,107 @@ namespace ivcam2 {
...
+        {
+            _active_dir = get_debug_path_base();
+            if( _active_dir.empty() )
+                return false;
+            add_dir_sep( _active_dir );
+
+#ifdef _WIN32
+            auto status = _mkdir( _active_dir.c_str() );
+#else
+            auto status = mkdir( dir.c_str(), 0700 );
```
Comment (Nir-Az) [inline src/l500/ac-trigger.cpp]: Can you add a comment about what 0700 means? I spend 2 minutes on google and still could not find the meaning..

**Review by Nir-Az — COMMENTED** [review]
Looks OK, gave 2 minor comments.

File: src/l500/ac-trigger.cpp (hunk 2 above)
Comment (maloel) [reply src/l500/ac-trigger.cpp]: // 0700 = user r/w/x

File: src/l500/ac-trigger.cpp (hunk 1 above)
Comment (maloel) [reply src/l500/ac-trigger.cpp]: Good idea. Done.

**Review by Nir-Az — APPROVED** [review]

**Pull request merged**